packages/ckb-indexer/src/collector.ts

```diff
@@ -154,6 +158,10 @@ export class CKBCellCollector implements BaseCellCollector {
         ];
       }
     }
+
+    if (!this.queries.scriptLenRange && this.queries.type === "empty") {
```

Comment:
Not sure if this one is necessary, is there any error if we don't do this change?

Reply:
> Not sure if this one is necessary, is there any error if we don't do this change?

this makes filter works in ckb-indexer